elephant/spike_train_generation.py

```diff
@@ -186,21 +186,34 @@ def spike_extraction(
         Default: None
     interval : tuple of :class:`pq.Quantity`
         Specifies the time interval around the `time_stamps` where the waveform
-        is extracted.
+        is extracted. The default values [-2ms, 4ms] are based on experience,
+        and many spike sorting tools choose values in this range:
+        Spikes are typically about 1-2 ms in length. This is to include a small interval before
```

Comment:
I think it would read better by making a continuous sentence, e.g., "choose values in this range as the spikes are typically ..."